src/lib/isMobilePhone.js

```diff
@@ -6,6 +6,7 @@ const phones = {
   'ar-AE': /^((\+?971)|0)?5[024568]\d{7}$/,
   'ar-BH': /^(\+?973)?(3|6)\d{7}$/,
   'ar-DZ': /^(\+?213|0)(5|6|7)\d{8}$/,
+  'ar-DJ': /^((\+|00)?253)?[7][7][678]\d{5}$/,
```

Comment:
Since the first part is not a range you can rewrite it that way
```suggestion
  'ar-DJ': /^((\+|00)?253)?77[678]\d{5}$/,
```

Reply:
Sure, will make the changes, thanks for the input 😀.